packages/grid/x-data-grid/src/tests/selection.DataGrid.test.tsx

```diff
@@ -346,48 +346,48 @@ describe('<DataGrid /> - Selection', () => {
     });
   });
 
+  /* eslint-disable material-ui/disallow-active-element-as-key-event-target */
```

Comment:
Could you update the tests so we don't need to ignore this ESLint rule? To keep consistent with the core.

Reply:
Done